R/pkg/R/generics.R

```diff
@@ -623,6 +623,10 @@ setGeneric("getItem", function(x, ...) { standardGeneric("getItem") })
 
 #' @rdname column
 #' @export
+setGeneric("isNaN", function(x) { standardGeneric("isNaN") })
```

Comment:
It won't uniform the ```isNaN``` and ```isnan``` interface at SQL side, please refer the comments at #10056. Further more, there are different semantics between Scala and Java about ```isnull```. So let's narrow this PR to provide the same functions as Scala at SparkR side, this is my original motivation. I will add test cases for column as @sun-rui suggested. Then I think this PR can be merged firstly and we start to discuss corresponding alias or explanations about the ```NULL/NA``` difference. @sun-rui @felixcheung @shivaram